### ISSUE TITLE
Beheading Geese is time consuming

### ISSUE THREAD

**Issue by DiscardedSlinky** [jasonrohrer/OneLife#993]
People dig up tree stumps in town because they're useless so there's usually maybe 1 or 2 around if you're lucky. Having to stand around and wait for the heads to despawn PER GOOSE is very time consuming if you're trying to make a life farming them. 20 seconds each. 

Maybe despawn faster? Or let us remove the heads again? 

It's not a HUGE problem just one I run into a lot because it's my favorite job. I've wasted many minutes standing around waiting for heads to despawn.

**Comment by TwistedHBG** [comment]
Just being able to click the stump to despawn the head would be great, we don't even need to have a pickupable goose head object as it doesn't serve a purpose

**Comment by Shadie116** [comment]
what do i know i'm in donkey town

**Comment by DougLefelhocz** [comment]
I agree that the head takes too long to despawn from the stump.  Though it would be realistic and probably better if it didn't spawn on a stump (only on the ground or trashpit) and it had to get removed from the stump manually.

Other than that, I think it's ok.  Geese beheading is kind of difficult to setup up and requires speed to do, but it makes sense that the geese would flee from the stump, and it's fine in terms of difficulty overall.
